src/main/java/com/sprint/mission/discodeit/controller/AuthController.java

```diff
@@ -0,0 +1,23 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.dto.LoginRequest;
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.service.AuthService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+@RequiredArgsConstructor
+@RequestMapping("/api/auth")
+public class AuthController {
+    private final AuthService authService;
+
+    @RequestMapping(method = RequestMethod.POST)
```

Comment:
로그인이라는 것을 특정할 수 있는 url suffix가 필요할 것 같아요.
```suggestion
    @RequestMapping(method = RequestMethod.POST, value = "/login")
```